indigo_app/templates/indigo_api/work_form.html

```diff
@@ -229,23 +229,34 @@ <h5 class="card-header">Commencement and Assent</h5>
       <div class="card-body">
 
         <div class="form-row">
-          <div class="form-group col-md-4">
-            <label for="{{ form.commencement_date.id_for_label }}">Commencement date</label>
-            <input type="text" class="form-control" data-provide="datepicker" id="{{ form.commencement_date.id_for_label }}" name="{{ form.commencement_date.html_name }}" placeholder="yyyy-mm-dd" pattern="\d{4}-\d\d-\d\d" value="{{ form.commencement_date.value|date:"Y-m-d"|default:'' }}">
-            <p class="form-text text-muted">Date when most of the work comes into force<br>(only leave blank if not yet commenced)</p>
-            {% if work.pk and work.commencement_date %}
-              {% if work.amendments_made.exists or work.repealed_works.exists %}
-                <div class="alert alert-warning">If you change this commencement date, you may also have to manually change the dates for related <a href="{% url 'work_related' work.frbr_uri %}">amendments and repeals</a> made by this work.</div>
-              {% endif %}
-            {% endif %}
+          <div class="form-group">
+            <div class="form-check">
+              <input type="checkbox" class="form-check-input" id="{{ form.commenced.id_for_label }}" name="{{ form.commenced.html_name }}" value="on" {% if form.commenced.value %}checked{% endif %}>
+              <label class="ml-2 mr-2" for="{{ form.commenced.id_for_label }}">Commenced – this work has come into force</label>
+            </div>
           </div>
+          <div id="commencement_alert" class="alert alert-warning">If you clear this and Save, the commencement date and/or work will be removed.</div>
```

Comment:
I'm not sure this is necessary. The entry fields disappear when it's unchecked, which I think makes it fairly obvious. Additionally, it's not a common operation to uncheck this once details have been entered.

Reply:
Agree it shouldn't be common; I wanted to warn people in case they think that un/checking the box just shows/hides the commencement details (because if you re-check it the details still show up)

Reply:
to be clear, it only shows up in specific circumstances:
 https://github.com/laws-africa/indigo/pull/849/files#diff-eecce2e94e31cbeb0cb999e041d7a824R233-R239

Reply:
i.e. when there are commencement details and the box has been unchecked

Reply:
that's not what it does, but it should be! fix incoming

Reply:
I still don't feel it's necessary. The options disappear, which I think is a clear indication that they're going to "go away" -- ie. be cleared. I feel the warning just makes it more confusing.

If the concern is that, after unchecking and then rechecking, the details are still there, then perhaps clear them instead? Like we do for removing the repeal.